### ISSUE TITLE
incorrectly escaped description of an update

### ISSUE THREAD

**Issue by jhutar** [fedora-infra/bodhi#1656]
Hello,
I have just submitted this update (note that is contains `<dcallagh@redhat.com>` in the description:

```
$ fedpkg update
/usr/lib/python2.7/site-packages/fedora/client/bodhi.py:48: DeprecationWarning: fedora.client.bodhi has been deprecated. Please use bodhi.client.bindings instead.
  DeprecationWarning)
Creating a new update for  python-rpmfluff-0.5.3-1.fc25 
================================================================================
     python-rpmfluff-0.5.3-1.fc25
================================================================================
  Update ID: FEDORA-2017-7e594f96bb
    Release: Fedora 25
     Status: pending
       Type: enhancement
      Karma: 0
    Request: testing
      Notes: Support mixing noarch and archful packages in a yum repo build by
           : Dan Callaghan <dcallagh@redhat.com>  ----  Lots of
           : fixes by Dan Callaghan <dcallagh@redhat.com>
  Submitter: jhutar
  Submitted: 2017-06-28 08:04:44
   Comments: bodhi - 2017-06-28 08:04:44 (karma 0)
             This update has been submitted for testing by jhutar.
             bodhi - 2017-06-28 08:04:50 (karma 0)
             This update has obsoleted [python-rpmfluff-0.5.2-1.fc2
             5](https://bodhi.fedoraproject.org/updates/FEDORA-2016
             -914adc225c), and has inherited its bugs and notes.

  https://bodhi.fedoraproject.org/updates/FEDORA-2017-7e594f96bb
```

Looking at the web there is `&lt;dcallagh@redhat.com>` and the link leads to invalid URL

**Comment by vedsarkushwaha** [comment]
I think I can fix this.
Is there any way to reproduce this issue.

**Comment by vedsarkushwaha** [comment]
Understood.
https://bodhi.fedoraproject.org/updates/FEDORA-2017-7e594f96bb
URL needs to be fixed.

How can I edit the HTML?

**Comment by bowlofeggs** [comment]
Hello @vedsarkushwaha - the update template is here: https://github.com/fedora-infra/bodhi/blob/develop/bodhi/server/templates/update.html

**Comment by vedsarkushwaha** [comment]
thanks @bowlofeggs - After making changes, how can I test it?

**Comment by bowlofeggs** [comment]
@vedsarkushwaha If you use the Bodhi development environment, you should be able to browse to http://localhost:6543/updates/FEDORA-2017-7e594f96bb and see the change. You can read about how to set up the Bodhi development environment here:

https://bodhi.fedoraproject.org/docs/developer_docs.html

**Comment by ryanlerch** [comment]
I *think* this escaping is the result of two things to do with the markdown support. WE use bleach to sanitize markdown input from the user before we parse the markdown. Bleach is used here to strip out _all_ tags before passing it to the markdown parser.

Our markdown parser also has a regex in it that linkifies some things, which i *think* might be adding to our issues here as well.

Note that you can re-create and test this easily too by just drafting a comment to an update, and checking the preview to see the incorrectly escaped result.

**Comment by vedsarkushwaha** [comment]
According to wiki, there are 2 ways to setup bodhi. None of them works for me.
Following command fails while following the steps:
For the Method 1:
cp Vagrantfile.example Vagrantfile says "Vagrantfile.exmple doesn't exist". Infact there was no Vagrantfile.example in the directory.

For the Method 2:
"alembic upgrade head" throws error "ImportError: No module named cornice.validators"

**Comment by bowlofeggs** [comment]
@vedsarkushwaha ```Vagrantfile.example``` is part of the repo:

https://github.com/fedora-infra/bodhi/blob/develop/Vagrantfile.example

The error message you quoted has a typo in it ("exmple" instead of "example") - is it possible that is the problem?

As for the cornice error - what version of cornice do you have? Bodhi doesn't work with Cornice 2 yet, perhaps you are on a newer version? I'm not sure if others are using the second development environment or not and I've not tried it myself, so it's possible the instructions are stale.

**Comment by bowlofeggs** [comment]
The patch for this issue should be included in the upcoming 2.10.0 release:

https://github.com/fedora-infra/bodhi/pull/1744
